webapi/src/main/scala/org/knora/webapi/responders/admin/ProjectsService.scala

```diff
@@ -77,19 +84,48 @@ final case class ProjectsServiceLive(bridge: ActorToZioBridge) extends ProjectsS
    *     '''failure''': [[dsp.errors.NotFoundException]] when no project for the given IRI can be found
    *                    [[dsp.errors.ForbiddenException]] when the user is not allowed to perform the operation
    */
-  def deleteProject(projectIri: IRI, requestingUser: UserADM): Task[ProjectOperationResponseADM] =
+  def deleteProject(projectIri: ProjectIri, requestingUser: UserADM): Task[ProjectOperationResponseADM] = {
+    val projectStatus = Project.ProjectStatus.make(false).getOrElse(throw BadRequestException(""))
     for {
       random      <- ZIO.random
       requestUuid <- random.nextUUID
       response <- bridge.askAppActor[ProjectOperationResponseADM](
                     ProjectChangeRequestADM(
                       projectIri = projectIri,
-                      changeProjectRequest = ChangeProjectApiRequestADM(status = Some(false)),
+                      projectUpdatePayload = ProjectUpdatePayloadADM(status = Some(projectStatus)),
                       requestingUser = requestingUser,
                       apiRequestID = requestUuid
                     )
                   )
     } yield response
+  }
+
+  /**
+   * Updates a project
+   *
+   * @param projectIri           the [[IRI]] of the project
+   * @param payload              a [[ProjectUpdatePayloadADM]] instance
+   * @param requestingUser       the user making the request.
+   * @return
+   *     '''success''': information about the project as a [[ProjectOperationResponseADM]]
+   *
+   *     '''failure''': [[dsp.errors.NotFoundException]] when no project for the given IRI can be found
+   */
+  def changeProject(
```

Comment:
`ProjectUpdatePayloadADM` vs the method name `changeProject`. 
I think we should aim to keep the naming consistent. If the necessity exists to use different names I would like to limit the inconsistency in naming in the api/rest layer if possible.
wdyt?

Reply:
I totally agree, I planned to only use "update" in the new code but this one slipped through.. I adjusted the method name.